client/freeShipping.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const FreeShipping = ({freeShipping}) => {
+  let shippingOffered = <div className="freeShipping">Free shipping to <span>United States</span></div>
+  let shippingNotOffered = <div className="freeShipping"></div>
+
+  if (freeShipping) {
+    return shippingOffered;

```

Comment:
```js
return (
<div className="freeShipping">
{ freeShipping && <span> Free Shipping to <span>United States</span></span>}
</div>
);
```
https://reactjs.org/docs/conditional-rendering.html 
You can take a look at the Inline If with Logical && Operator. Since all react components are true, if it isn't free shipping, Free shipping to <span>United States</span> won't show.

Reply:
cool didn't know about this